File: frontend/src/app/models/Food.ts
export interface Food {
  uuid?: string;
  name: string;
  energy: number;
  carbs: number;
  protein: number;
  fat: number;
  fiber?: number;
  sugar?: number;
  username?: string;
  amount?: number;
  servingSize?: number;
  packageSize?: number;
  editing?: boolean;
  reasonForEditing?: string;
  en?: string;
  waitingForApproval?: boolean;
}

File: rest-api/Models/Food.cs
using System;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using System.Collections.Generic;
namespace Makro.Models
{
    public class Food
    {
        [JsonIgnore]
        public int Id { get; set; }
        [Required]
        public string UUID { get; set; }
        [Required]
        public User User { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public decimal Energy { get; set; }
        [Required]
        public decimal Protein { get; set; }
        [Required]
        public decimal Carbs { get; set; }
        [Required]
        public decimal Fat { get; set; }
        public decimal Sugar { get; set; }
        public decimal Fiber { get; set; }
        public decimal PackageSize { get; set; }
        public decimal ServingSize { get; set; }
        public string En { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        [JsonIgnore]
        public ICollection<MealFood> MealFoods { get; set; }
        [JsonIgnore]
        public ICollection<SharedMealFood> SharedMealFoods { get; set; }

    }
}
